library/core/src/intrinsics/mod.rs

```diff
@@ -328,17 +320,13 @@ pub const fn prefetch_write_data<T, const LOCALITY: i32>(data: *const T) {
 /// This intrinsic does not have a stable counterpart.
 #[rustc_intrinsic]
 #[rustc_nounwind]
-<<<<<<< HEAD
-#[cfg(not(feature = "ferrocene_certified"))]
-pub unsafe fn prefetch_read_instruction<T>(data: *const T, locality: i32);
-=======
 #[miri::intrinsic_fallback_is_spec]
+#[cfg(not(feature = "ferrocene_certified"))]
 pub const fn prefetch_read_instruction<T, const LOCALITY: i32>(data: *const T) {
```

Comment:
It's really neat how a lot of these functions lost their `unsafe` due to `const`?